apps/condo/migrations/20231213105633-0352_auto_20231213_0556.js

```diff
@@ -0,0 +1,134 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;    
+--
+-- [CUSTOM] Set Statement Timeout to some large amount - 25 min (25 * 60 => 1500 sec)
+--
+SET statement_timeout = '1500s';
+
+--
+-- Add field canManageMarketplace to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canManageMarketplace" boolean DEFAULT false NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canManageMarketplace" DROP DEFAULT;
+--
+-- Add field canReadMarketplace to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canReadMarketplace" boolean DEFAULT false NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canReadMarketplace" DROP DEFAULT;
+--
+-- Add field canReadPaymentsWithInvoices to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canReadPaymentsWithInvoices" boolean DEFAULT false NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canReadPaymentsWithInvoices" DROP DEFAULT;
+--
+-- Add field canManageMarketplace to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canManageMarketplace" boolean NULL;
+--
+-- Add field canReadMarketplace to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canReadMarketplace" boolean NULL;
+--
+-- Add field canReadPaymentsWithInvoices to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canReadPaymentsWithInvoices" boolean NULL;
+
+--
+-- [CUSTOM] Set marketplace permissions to existed roles
+--
```

Comment:
What changes are there for the Master role?

Reply:
`employee.role.Foreman.name`
str number 66

Reply:
oops, oh that English :)